test/blocks/index.js

```diff
@@ -0,0 +1,73 @@
+import mocha from 'mocha';
+import chai, { expect } from 'chai';
+import dirtyChai from 'dirty-chai';
+
+import * as blocks from '@wordpress/blocks';
```

Comment:
In this particular case, it's not making any difference, but worth noting that in other use cases when importing from external libraries, it can have an impact on `tree shaking` 

```js
import * as _ from 'lodash'; console.log( _.uniqueId() ); // This will include the whole lodash
import { uniqueId } from 'lodash'; console.log( uniqueId() ); // This will include only uniqueId
```